### PR TITLE
refactor: timeoutBlockNumber is not hardcoded

### LINKED ISSUE #84: Different timeouts for each chain
The HTLCs for the Bitcoin and Litecoin chain should have different timeouts and it should be made sure that the expiry date of the invoice is higher than the expected timeout of the onchain HTLC

### PR DESCRIPTION
**Pull request by ImmanuelSegol**
Closes #84

### CODE REVIEW

File: lib/Config.ts [hunk 1]
```diff
@@ -78,6 +78,8 @@ class Config {
       {
         symbol: 'BTC',
 
+        timeoutBlockNumber: 10,
```
Comment (michael1011) [inline lib/Config.ts]: The default value for BTC should be `2`

File: lib/Config.ts [hunk 2]
```diff
@@ -89,6 +91,8 @@ class Config {
       {
         symbol: 'LTC',
 
+        timeoutBlockNumber: 10,
```
Comment (michael1011) [inline lib/Config.ts]: And `8` for LTC

File: lib/consts/Types.ts [hunk 3]
```diff
@@ -14,6 +14,8 @@ export type SwapUpdate = {
 export type CurrencyConfig = {
   symbol: string;
 
+  timeoutBlockNumber: number;
```
Comment (michael1011) [inline lib/consts/Types.ts]: Let's call this variable `timeoutBlockDelta` instead. That might be less confusing and irritating

File: lib/service/Service.ts [hunk 4]
```diff
@@ -420,6 +435,13 @@ class Service extends EventEmitter {
     });
   }
 
+  private getChainConfig = (symbol: string): CurrencyConfig | undefined => {
```
Comment (michael1011) [inline lib/service/Service.ts]: In the long run initializing a map with the symbol as key in the `constructor` might be better. What do you think?

File: lib/service/Service.ts [hunk 5]
```diff
@@ -223,6 +225,12 @@ class Service extends EventEmitter {
 
     const { base, quote, rate } = this.getPair(pairId);
 
+    const chainConfig = this.getChainConfig(base);
+
+    if (!!chainConfig) {
```
Comment (michael1011) [inline lib/service/Service.ts]: I would prefer `chainConfig !== undefined`. This style sill confuses me every time I read it.

File: lib/service/Service.ts (hunk 4 above)
Comment (ImmanuelSegol) [reply lib/service/Service.ts]: Yeah, we could do that but I think this would be fine for now.

**Comment by ImmanuelSegol** [comment]
@michael1011  Sorry didn't notice the test where failing

**Pull request merged**